libc/src/__support/OSUtil/windows/exit.cpp

```diff
@@ -0,0 +1,19 @@
+//===----------- Windows implementation of an exit function -----*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/__support/macros/config.h"
+
+#include <Windows.h>
```

Comment:
would it be better to surround it with #pragma push_macro “WIN32_LEAN_AND_MEAN?

Reply:
Is the `push_macro` and `pop_macro` pragmas necessary? I added `WIN32_LEAN_AND_MEAN` only.

Reply:
I think it is fine. This is a `.cpp` file so adding `WIN32_LEAN_AND_MEAN` is sufficient.